src/test/java/picard/analysis/CollectAlignmentSummaryMetricsTest.java

```diff
@@ -768,4 +768,37 @@ public void testReadLengthHistogram(final boolean plotChart) throws IOException
             }
         }
     }
+
+    @Test
+    public void testNoPFReads() throws IOException {
```

Comment:
In this case, do you want an IOException if there are no PF reads? I guess it's OK to have an exception in this case, since it seems pointless to try to collect metrics on a file with no passing alignments, but maybe make it a `PicardException` with a useful error message (this could be done instead of my warning comment before).